src/rollback_to_stable/rts_btree.c

```diff
@@ -321,9 +321,7 @@ __rts_btree_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_REF *ref, WT_ROW *rip,
      * have moved this value to the history store as a full value. Therefore, we can safely ignore
      * the on page value if it is overflow removed.
      */
-    if (__wt_cell_type_raw(unpack->cell) == WT_CELL_VALUE_OVFL_RM)
-        ret = 0;
-    else
+    if (__wt_cell_type_raw(unpack->cell) != WT_CELL_VALUE_OVFL_RM)
```

Comment:
This is a functional change, the `ret = 0` was a weird and seemed like a dead store. @agorrod you implemented this so there's a chance you'd know why it was done this way?

Reply:
From Alex:
It was added by Chenhao here:
https://github.com/wiredtiger/wiredtiger/commit/e989c881bfe90086b2341aa8837739f53b0f24d2
(I just moved it). I don't mind getting rid of artifacts in return values that might cause problems in the future (e.g if the next WT_ERR is removed from this function, that carried forward ret might cause an error to be returned accidentally.
OTOTH: I don't have strong feelings.